doc/Comprobaciones/0_Requisitos.md

```diff
@@ -0,0 +1,26 @@
+# :bookmark_tabs: Lista de comprobación: 
+
+- :heavy_check_mark: ¿Se trata de un problema real del que se tenga conocimiento personal? <br/>
+    
+    Si, para poder trabajar y entrenar a una IA con deep learning se usarán datos de un csv recogido de los incendios ocurridos en la zona norte de [Portugal](http://archive.ics.uci.edu/ml/datasets/Forest+Fires). Con estos datos se pueden verificar si realmente están bien hecho las operaciones. 
```

Comment:
Piénsate bien lo de la IA con deep learning. Insisto, *tenéis que programarlo vosotros* y preparar los tests de forma que comprueben que se hace la predicción correctamente.